astropy/modeling/tests/test_blackbody.py

```diff
@@ -32,8 +32,8 @@ def test_evaluate(self, temperature):
         b = BlackBody1D(temperature=temperature,
                         bolometric_flux=bolometric_flux)
 
-        assert_quantity_allclose(b(1.4 * u.micron), 4756726.111003904 * u.Jy)
-        assert_quantity_allclose(b(214.13747 * u.THz), 4756726.111003904 * u.Jy)
+        assert_quantity_allclose(b(1.4 * u.micron), 4734464.498937388 * u.Jy)
```

Comment:
The relative change is 4.7e-3 which is due to the same relative change in L_sun. It's because the blackbody is normalized to `bolometric_flux = 1000 * u.L_sun / (4 * np.pi * (1.5 * u.pc)**2)`.